frontend/package.json

```diff
@@ -4,10 +4,11 @@
   "description": "Table Soccer Tournaments React Web App",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.11.3",
+    "@material-ui/icons": "^4.11.2",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
-    "@types/jest": "^26.0.15",
```

Comment:
Dlaczego usunięte?

Reply:
Tak ja wyżej, próbowałam rozwiązać problem z niedziałającym testem do komponentu Footouer. To było jedno z zaproponowanych rozwiązań